woocommerce/woocommerce
# Add launch your store homescreen status tour

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #46070  .

This PR adds a tour for the homescreen site status badge.

![Screen Shot 2024-04-03 at 3 39 34 PM](https://github.com/woocommerce/woocommerce/assets/4723145/c9b0aadb-50d0-466d-bcfd-6393a8117116)




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new JN site with `launch-your-store` feature flag enabled.
2. Go to `WooCommerce -> Home`
3. Confirm the tour doesn't show up.

Updating from older versions of WooCommerce

1. Create a new JN site without WooCommerce.
2. Download the latest 8.6.x version [here](https://downloads.wordpress.org/plugin/woocommerce.8.6.1.zip)
3. Upload the zip file and activate it.
4. Checkout this branch locally and build a new zip file in `plugins/woocommerce` directory with `pnpm run build:zip` command
5. Upload the zip file and replace the current version with it.
6. A database update notice should show up. Click on update.
7. SSH into the server and run `wp option get woocommerce_show_lys_tour`
8. Confirm the option exists.
9. Go to `WooCommerce -> Home`
10. Confirm the tour shows up.
11. Close the tour
12. Refresh the page
13. Confirm the tour is not shown



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Add a tour for the homescreen site status badge.

#### Comment 

</details>
